TicTacToeApp/ViewController.swift

```diff
@@ -21,25 +20,45 @@ public class ViewController: UIViewController {
     // Dispose of any resources that can be recreated.
   }
 
-  @IBOutlet public var boardButtons: [UIButton]!
-  @IBOutlet weak var resetButton: UIButton!
-
   @IBAction public func resetBoard() {
-    boardButtons.forEach{ button in
-      button.setTitle(nil, forState: UIControlState.Normal)
-      button.enabled = true
-    }
-    currentPlayerMark = "X"
+    clearBoard()
+    enableBoardButtons()
+    game = Game()
+    updateGameInformationDisplay()
   }
 
   @IBAction public func makeMove(button: UIButton) {
-    button.setTitle(currentPlayerMark, forState: UIControlState.Normal)
-    currentPlayerMark = toggleMark(currentPlayerMark)
+    updateGameBoardButtonForPlayerMove(button, playerMark: game.getCurrentPlayer())
+    game.makeMove(button.tag)
+    updateGameInformationDisplay()
+
+    if game.playerWonLastTurn(game.getInactivePlayer()) {
+      disableBoardButtons()
+    }
+  }
+
+  private func updateGameInformationDisplay() {
+    infoLabel.text = gamePrompt.promptFor(game)
+  }
+
+  private func updateGameBoardButtonForPlayerMove(button: UIButton, playerMark: PlayerMark) {
+    button.setTitle(getButtonTextFor(playerMark), forState: UIControlState.Normal)
     button.enabled = false
   }
 
-  private func toggleMark(mark: String) -> String {
-    return mark == "X" ? "O" : "X"
+  private func clearBoard() {
+    boardButtons.forEach{ $0.setTitle(nil, forState: UIControlState.Normal) }
+  }
+
+  private func enableBoardButtons() {
+    boardButtons.forEach{ $0.enabled = true }
+  }
+
+  private func disableBoardButtons() {
+    boardButtons.forEach{ $0.enabled = false }
   }
-}
 
+  private func getButtonTextFor(playerMark: PlayerMark) -> String {
+    return playerMark == PlayerMark.X ? "X" : "O"
```

Comment:
Hypothetical question: Which places would you need to change if you wanted to display images instead of strings for the player marks?

If the answer involves one or more of your "core" classes, then you have your view coupled to your logic.

In this case, I _think_ you are OK with using the `rawValue`, since you seem to be only referencing `rawValue` in view related classes.

Alternatively, if changing how you interpolate the player mark for the prompt would force a change to updating the button text (or vice versa), then you may have coupling.